packages/nodes-base/nodes/Merge/test/v3/operations.test.ts

```diff
@@ -259,6 +259,142 @@ describe('Test MergeV3, combineBySql operation', () => {
 			country: 'PL',
 		});
 	});
+
+	it('SELECT query, should run executeSelectWithMapedPairedItems and collect pairedItems data', async () => {
```

Comment:
Here the test naming convention is `it` in the file below it's `test`. 

**Out of Scope** discuss with team but removing `should` and conjugue present tense reduces test titles by 5 letters.

```ts
describe(`SELECT query`,()=>{
  it(`runs executeSelectWithMapedPairedItems and collects pairedItems data`,()=>{});
})
```